pkg/sfu/buffer/buffer.go

```diff
@@ -324,6 +326,7 @@ func (b *Buffer) Write(pkt []byte) (n int, err error) {
 	b.payloadType = rtpPacket.PayloadType
 	b.calc(pkt, &rtpPacket, time.Now(), false)
 	b.Unlock()
+	b.readCond.Signal()
```

Comment:
signal should be inside lock?

Reply:
nvm 

> It is allowed but not required for the caller to hold c.L during the call.

Reply:
doc said `It is allowed but not required for the caller to hold c.L during the call.`